jobs/Reservations/readme.md

```diff
@@ -0,0 +1,34 @@
+# Reservation Management
+
+**Fullstack App**
+
+> [!NOTE]
+> This task will involve a separate repository.
+
+Start by forking [the mews-reservations-int repo](https://github.com/MewsSystems/reservations-interview) and
+following the "Running the App" portion of the readme.
+
+```sh
+git clone git@github.com:MewsSystems/reservations-interview.git
+```
+
+You will be acting as if you are the lead engineer on a team, executing tasks from the `/backlog`. You are free to pick up
```

Comment:
If someone is a mid-level engineer, could the `lead engineer` element here throw them?  Why would we not want them to just perform it at their current level?

Reply:
I see what you mean - I can remove the hinted level and instead lean on what our current tasks do, which is give freedom for the candidate to use/leverage whatever tech/libraries they want.

Reply:
Done in ca10a09.